packages/vuetify/src/components/VCarousel/__tests__/VCarousel.spec.ts

```diff
@@ -125,4 +124,57 @@ describe('VCarousel.ts', () => {
 
     expect(wrapper.vm.internalHeight).toBe(300)
   })
+
+  it('should have the correct theme', async () => {
+    let wrapper = mountFunction({}, true)
+
+    expect(wrapper.vm.isDark).toBeFalsy()
+    expect(wrapper.html()).toMatchSnapshot();
```

Comment:
i think you could just check existence of `.v-carousel.theme-light/dark` element instead of generating the whole markup, not sure what are team's thoughts on this

Reply:
Yep, I just wanted to see the html generated to make sure, whatever is preferred